charts/bindplane/README.md

```diff
@@ -91,6 +91,7 @@ BindPlane OP is an open source observability pipeline.
 | eventbus.pubsub.projectid | string | `""` |  |
 | eventbus.pubsub.topic | string | `""` |  |
 | eventbus.type | string | `""` |  |
+| extraEnv | list | `[]` | Optional arbitrary environment variables to add to the BindPlane pod(s). |
```

Comment:
This should have been added in a previous pr, we must of forgot to regen the docs.